### PR TITLE
Implement CassInet and related functions

### PR DESCRIPTION
**Pull request by constantfold**
Implement `CassInet`, all `cass_inet*` functions and statement binding of `CassInet`.

### CODE REVIEW

File: scylla-rust-wrapper/src/inet.rs [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+    let input = ptr_to_cstr_n(input_raw, input_length);
+    if input.is_none() {
+        return crate::cass_error::LIB_BAD_PARAMS;
+    }
+
+    let input_str = input.unwrap();
+    let ip_addr = IpAddr::from_str(input_str);
+
+    match ip_addr {
+        Result::Ok(ip_addr) => {
```
Comment (piodul) [inline scylla-rust-wrapper/src/inet.rs]: No need to use qualified `Result::Ok` or `Result::Err`, those are imported by Rust by default. Use `Ok` and `Err` instead.

File: scylla-rust-wrapper/src/inet.rs [hunk 2]
```diff
@@ -0,0 +1,144 @@
+use crate::argconv::*;
+use crate::cass_error::CassError;
```
Comment (piodul) [inline scylla-rust-wrapper/src/inet.rs:2]: You can change it to the following:

```rust
use crate::cass_error::{self, CassError};
```

Then you will be able to change occurrences of `crate::cass_error::ERROR_NAME` to `cass_error::ERROR_NAME` in this file.

**Review by piodul — APPROVED** [review]

File: scylla-rust-wrapper/src/inet.rs (hunk 1 above)
Comment (constantfold) [reply scylla-rust-wrapper/src/inet.rs]: I was just too cautious as when I tried this:

```
enum MyEnum {
A, B
}
...
match(my_enum_instance) {
  A => {},
  B => {},
}
```

it did not do what I expect and had to write `MyEnum::A`, `MyEnum::B` in the `match`.

File: scylla-rust-wrapper/src/inet.rs (hunk 2 above)
Comment (constantfold) [reply scylla-rust-wrapper/src/inet.rs:2]: Purposefully did not make those occurrences "clean", as #12 changes the error type and all those instances will be changed either way after that PR is merged.

File: scylla-rust-wrapper/src/inet.rs (hunk 1 above)
Comment (constantfold) [reply scylla-rust-wrapper/src/inet.rs]: Fixed

**Pull request merged**